### PR TITLE
[DOCS-6867] Restructure OTel Collector Datadog Exporter docs

### PR DESCRIPTION
**Pull request by kayayarai**
### What does this PR do? What is the motivation?
DOCS-6867

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/opentelemetry/collector_exporter/trace_metrics.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+{{< img src="/opentelemetry/collector_exporter/trace_metrics.png" alt="APM metrics from OpenTelemetry" style="width:100%;" >}}
+
+To send APM stats such as hits, errors, and duration, set up the Datadog connector.
+
+For more information, see the OpenTelemetry project documentation for the [Datadog Connector][1].
+
+## Setup
+
+```yaml
```
Comment (cswatt) [inline content/en/opentelemetry/collector_exporter/trace_metrics.md:19]: context

File: content/en/opentelemetry/collector_exporter/trace_metrics.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    exporters: [datadog]
+```
+
+## Data collected
+
+See [Trace Metrics][2].
+
+## Full example configuration
+
+For a full working example, see the Datadog Exporter example in [`trace-metrics.yaml`][3].
```
Comment (cswatt) [inline content/en/opentelemetry/collector_exporter/trace_metrics.md:52]:
```suggestion
For a full working example configuration for the Datadog Exporter, see [`trace-metrics.yaml`][3].
```

File: content/en/opentelemetry/collector_exporter/hostname_tagging.md [hunk 3]
```diff
@@ -0,0 +1,431 @@
+---
+title: Datadog Exporter Hostname and Tagging Configuration
```
Comment (cswatt) [inline content/en/opentelemetry/collector_exporter/hostname_tagging.md:2]:
```suggestion
title: Datadog Exporter Collector Hostname and Tagging
```

File: content/en/opentelemetry/collector_exporter/log_collection.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
+---
+title: Datadog Exporter Docker Stats Receiver Configuration
```
Comment (cswatt) [inline content/en/opentelemetry/collector_exporter/log_collection.md:2]:
```suggestion
title: Datadog Exporter Collector Log Collection
```

File: content/en/opentelemetry/collector_exporter/otlp_receiver.md [hunk 5]
```diff
@@ -0,0 +1,189 @@
+---
+title: Datadog Exporter OTLP Receiver
```
Comment (cswatt) [inline content/en/opentelemetry/collector_exporter/otlp_receiver.md:2]:
```suggestion
title: Datadog Exporter Collector OTLP Receiver
```

File: content/en/opentelemetry/collector_exporter/trace_metrics.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
+---
+title: Datadog Exporter Trace Metrics
```
Comment (cswatt) [inline content/en/opentelemetry/collector_exporter/trace_metrics.md:2]:
```suggestion
title: Datadog Exporter Collector Trace Metrics
```

File: content/en/opentelemetry/collector_exporter/_index.md [hunk 7]
```diff
@@ -0,0 +1,222 @@
...
+       -p 4317:4317 \
+       --hostname $(hostname) \
+       -v $(pwd)/otel_collector_config.yaml:/etc/otelcol-contrib/config.yaml \
+       otel/opentelemetry-collector-contrib
+   ```
+
+
+[1]: https://hub.docker.com/r/otel/opentelemetry-collector-contrib/tags
+{{% /tab %}}
+{{% tab "Docker (other containers)" %}}
```
Comment (cswatt) [inline content/en/opentelemetry/collector_exporter/_index.md:84]: where does configuration go? i see an `otel_collector_config.yaml` and a `config.yaml` in step 2, is that related? if so, which one/both?

File: content/en/opentelemetry/collector_exporter/_index.md [hunk 8]
```diff
@@ -0,0 +1,222 @@
...
+
+
+
+[1]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/2c32722/exporter/datadogexporter/examples/k8s-chart
+[2]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/2c32722/exporter/datadogexporter/examples/k8s-chart/daemonset.yaml#L41-L46
+
+{{% /tab %}}
+{{% tab "Kubernetes (Gateway)" %}}
+
+To deploy the OpenTelemetry Collector and Datadog Exporter in a Kubernetes Gateway deployment, use this [full example of configuring the OpenTelemetry Collector using the Datadog Exporter as a DaemonSet][1], including the application configuration example.
```
Comment (cswatt) [inline content/en/opentelemetry/collector_exporter/_index.md:146]: is this meant to link to the same examples as for Kubernetes DaemonSet? if so, then DaemonSet and Gateway have exactly the same instructions. is that intentional?

**Review by cswatt — CHANGES_REQUESTED** [review]
My biggest concern is that I'm unclear on how the Datadog Exporter is related to the OpenTelemetry Collector. Is the exporter a component of the collector? And most critically—how does this play out wrt configuration? Do they share the same configuration? Is there a distinction between "configure the Exporter" and "configure the Collector"?

Other key points:
- The setup codeblocks on the component pages need some context ("add this to your config file") / explicit direction as to what exactly to do with the codeblocks provided
- On the Hostname and Tagging page, I can't really make sense of the Kubernetes DaemonSet->Gateway setup
- I'm dubious about the page titles... I made some suggestions but I am really not sure about them

And then some regular spag stuff.

File: content/en/opentelemetry/collector_exporter/_index.md [hunk 9]
```diff
@@ -0,0 +1,222 @@
...
+## Setting up the OpenTelemetry Collector with the Datadog Exporter
+
+To run the OpenTelemetry Collector along with the Datadog Exporter, download the latest release of the [OpenTelemetry Collector Contrib distribution][3]. 
+
+### Out-of-the-box Datadog Exporter configuration
+
+In the `exporter/datadogexporter` folder is a working example of an out-of-the-box configuration for Datadog Exporter. See the full configuration example file, [`ootb-ec2.yaml`][4]. Configure each of the following components to suit your needs:
+
+{{< whatsnext desc="Exporter components" >}}
+    {{< nextlink href="/opentelemetry/collector_exporter/hostname_tagging/" >}}Hostname and tags{{< /nextlink >}}
```
[the remark this answers is not in the document]
Comment (kayayarai) [reply content/en/opentelemetry/collector_exporter/_index.md:40]:
```suggestion
    {{< nextlink href="/opentelemetry/collector_exporter/hostname_tagging/" >}}Hostname and Tags{{< /nextlink >}}
```

File: content/en/opentelemetry/collector_exporter/_index.md [hunk 10]
```diff
@@ -0,0 +1,222 @@
...
+{{% /tab %}}
+
+{{% tab "Docker (localhost)" %}}
+To run the OpenTelemetry Collector as a Docker image and receive traces from the same host:
+
+1. Choose a published Docker image such as [`otel/opentelemetry-collector-contrib`][1].
+
+2. Determine which ports to open on your container so that OpenTelemetry traces are sent to the OpenTelemetry Collector. By default, traces are sent over gRPC on port 4317. If you don't use gRPC, use port 4318.
+
+3. Run the container and expose the necessary port, using the previously defined `collector.yaml` file. For example, considering you are using port 4317:
```
[the remark this answers is not in the document]
Comment (kayayarai) [reply content/en/opentelemetry/collector_exporter/_index.md:71]:
```suggestion
3. Run the container and expose the necessary port, using the `collector.yaml` file. For example, if you are using port 4317:
```

**Comment by kayayarai** [comment]
@cswatt 

> I'm unclear on how the Datadog Exporter is related to the OpenTelemetry Collector. Is the exporter a component of the collector? And most critically—how does this play out wrt configuration? Do they share the same configuration? Is there a distinction between "configure the Exporter" and "configure the Collector"?

There's a diagram here https://docs.datadoghq.com/opentelemetry/ that shows how they're related-- does that help? I think you're correct that we need to be more specific about where the configs go. I'll add that.

File: content/en/opentelemetry/collector_exporter/_index.md (hunk 7 above)
Comment (mackjmr) [reply content/en/opentelemetry/collector_exporter/_index.md:84]: Are you referring to `-v $(pwd)/otel_collector_config.yaml:/etc/otelcol-contrib/config.yaml` ? If so, this is to mount volumes, so the `otel_collector_config.yaml` on the machine will be available in the container at this path: `/etc/otelcol-contrib/config.yaml`.

File: content/en/opentelemetry/collector_exporter/collector_health_metrics.md [hunk 11]
```diff
@@ -0,0 +1,128 @@
...
+| `otelcol_scraper_errored_metric_points` | Number of metric points that were unable to be scraped |
+| `otelcol_scraper_scraped_metric_points` | Number of metric points successfully scraped |
+| `otelcol_receiver_refused_metric_points` | Number of metric points that could not be pushed into the pipeline |
+| `otelcol_receiver_accepted_metric_points` | Number of metric points successfully pushed into the pipeline |
+| `otelcol_process_runtime_heap_alloc_bytes` | Bytes of allocated heap objects (see 'go doc runtime.MemStats.HeapAlloc') |
+| `otelcol_process_runtime_total_alloc_bytes` | Cumulative bytes allocated for heap objects (see 'go doc runtime.MemStats.TotalAlloc') |
+| `otelcol_exporter_enqueue_failed_log_records` | Number of log records failed to be added to the sending queue |
+| `otelcol_processor_batch_timeout_trigger_send` | Number of times the batch was sent due to a timeout trigger |
+| `otelcol_exporter_enqueue_failed_metric_points` | Number of metric points failed to be added to the sending queue |
+| `otelcol_process_runtime_total_sys_memory_bytes` | Total bytes of memory obtained from the OS (see 'go doc runtime.MemStats.Sys') |
```
[the remark this answers is not in the document]
Comment (mackjmr) [reply content/en/opentelemetry/collector_exporter/collector_health_metrics.md:55]: Yes that is it 👍

File: content/en/opentelemetry/collector_exporter/docker_metrics.md [hunk 12]
```diff
@@ -0,0 +1,131 @@
...
+extraVolumes:
+ - name: docker-sock
+   hostPath:
+     path: /var/run/docker.sock
+extraVolumeMounts:
+ - name: docker-sock
+   mountPath: /var/run/docker.sock
+```
+
+Collector configuration:
```
[the remark this answers is not in the document]
Comment (mackjmr) [reply content/en/opentelemetry/collector_exporter/docker_metrics.md:67]: This is for the collector. The datadog exporter is also a collector configuration.

File: content/en/opentelemetry/collector_exporter/docker_metrics.md [hunk 13]
```diff
@@ -0,0 +1,131 @@
...
+{{% tab "Host" %}}
+
+The Docker stats receiver needs access to the Docker socket. By default, the receiver looks for the Docker socket at `unix:///var/run/docker.sock`. If this is not the Docker socket path, specify the path in the `endpoint` configuration line.
+
+**Note**: If you are using the collector image, you may need to [configure additional permissions for the collector to have access to the Docker socket][1]. 
+
+```yaml
+receivers:
+  docker_stats:
+    endpoint: unix:///var/run/docker.sock
```
Comment (mackjmr) [inline content/en/opentelemetry/collector_exporter/docker_metrics.md:30]:
```suggestion
    endpoint: unix:///var/run/docker.sock # (default)
```

File: content/en/opentelemetry/collector_exporter/docker_metrics.md [hunk 14]
```diff
@@ -0,0 +1,131 @@
...
+ - name: docker-sock
+   mountPath: /var/run/docker.sock
+```
+
+Collector configuration:
+
+```yaml
+receivers:
+  docker_stats:
+    endpoint: unix:///var/run/docker.sock
```
Comment (mackjmr) [inline content/en/opentelemetry/collector_exporter/docker_metrics.md:72]:
```suggestion
    endpoint: unix:///var/run/docker.sock # (default)
```

File: content/en/opentelemetry/collector_exporter/collector_batch_memory.md [hunk 15]
```diff
@@ -0,0 +1,90 @@
...
+  batch:
+    # Datadog APM Intake limit is 3.2MB.    
+    send_batch_max_size: 1000
+    send_batch_size: 100
+    timeout: 10s
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+```yaml
```
Comment (mackjmr) [inline content/en/opentelemetry/collector_exporter/collector_batch_memory.md:58]: I don't think this prometheus receiver should be here. The prometheus receiver is for the Collector Health Metrics sections

**Comment by mackjmr** [comment]
For the additional context comments: any config file mentioned in the docs refers to the OpenTelemetry Collector configuration, except where it says `values.yaml` in which case it refers to the OpenTelemetry Collector helm chart.

File: content/en/opentelemetry/collector_exporter/collector_health_metrics.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (kayayarai) [reply content/en/opentelemetry/collector_exporter/collector_health_metrics.md:55]: adding a link

File: content/en/opentelemetry/collector_exporter/otlp_receiver.md [hunk 16]
```diff
@@ -0,0 +1,191 @@
...
+title: OTLP Receiver
+further_reading:
+- link: "/opentelemetry/collector_exporter/"
+  tag: "Documentation"
+  text: "Setting Up the OpenTelemetry Collector"
+---
+
+## Overview
+
+To collect OTLP metrics, logs, and traces, configure the [OTLP receiver][1] in your Datadog Exporter.
```
Comment (dineshg13) [inline content/en/opentelemetry/collector_exporter/otlp_receiver.md:11]: Configure OTLP receiver in Collector.

File: content/en/opentelemetry/collector_exporter/otlp_receiver.md [hunk 17]
```diff
@@ -0,0 +1,191 @@
...
+       endpoint: "localhost:4317"
+```
+
+## Data collected
+
+Traces, metrics, and logs.
+
+## Full example configuration
+
+For a full working example configuration for the Datadog Exporter, see [`otlp.yaml`][2].
```
Comment (dineshg13) [inline content/en/opentelemetry/collector_exporter/otlp_receiver.md:35]: reword : `for the Datadog Exporter` to  `with the Datadog exporter` in all the pages.

File: config/_default/menus/menus.en.yaml [hunk 18]
```diff
@@ -731,11 +731,51 @@ main:
     pre: open-telemetry
     parent: essentials_heading
     weight: 46000
-  - name: OpenTelemetry Collector Datadog Exporter
-    url: opentelemetry/otel_collector_datadog_exporter/
-    identifier: otel_dd_exporter
+  - name: Set Up the Collector
```
Comment (dineshg13) [inline config/_default/menus/menus.en.yaml:734]: May be `Getting Started with Collector` ?

File: content/en/opentelemetry/collector_exporter/host_metrics.md [hunk 19]
```diff
@@ -0,0 +1,148 @@
...
+
+For more information, including supported operating systems, see the OpenTelemetry project documentation for the [host metrics receiver][1].
+
+
+## Setup
+
+{{< tabs >}}
+{{% tab "Host" %}}
+
+Add the following lines to your Datadog Exporter configuration:
```
Comment (dineshg13) [inline content/en/opentelemetry/collector_exporter/host_metrics.md:23]: `Datadog Exporter` - `Collector`

File: content/en/opentelemetry/collector_exporter/otlp_receiver.md [hunk 20]
```diff
@@ -0,0 +1,191 @@
...
+
+## Overview
+
+To collect OTLP metrics, logs, and traces, configure the [OTLP receiver][1] in your Datadog Exporter.
+
+For more information, see the OpenTelemetry project documentation for the [OTLP receiver][1].
+
+## Setup
+
+Add the following lines to your Datadog Exporter configuration:
```
Comment (dineshg13) [inline content/en/opentelemetry/collector_exporter/otlp_receiver.md:17]: Collector configuration.

**Review by cswatt — APPROVED** [review]
